src/component/Background/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledBackground = styled.div`
+  height: 1220px;
+  border: none;
+  background-color: #f9f9f9;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 50px;
+`;
+
+const Background = (props) => {
+  return <StyledBackground>{props.children}</StyledBackground>;
+};
+
+export default Background;
```

Comment:
```suggestion
const Background = styled.div`
  height: 1220px;
  border: none;
  background-color: #f9f9f9;
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  padding: 50px;
`;

export default Background;
```

위와 같이 축약해볼 수도 있겠네요 👍